src/ext/java/org/opentripplanner/ext/traveltime/TravelTimeResource.java

```diff
@@ -213,16 +233,33 @@ public Response getSurface() {
   }
 
   private ZSampleGrid<WTWD> getSampleGrid() {
-    final RoutingRequest accessRequest = routingRequest.clone();
-
-    accessRequest.maxAccessEgressDuration = traveltimeRequest.maxAccessDuration;
-
-    try (var temporaryVertices = new TemporaryVerticesContainer(graph, accessRequest)) {
-      final Collection<AccessEgress> accessList = getAccess(accessRequest, temporaryVertices);
+    final NewRouteRequest accessRequest = routingRequest.clone();
+    // TODO: 2022-08-25 clone
+    final RoutingPreferences accessPreferences = routingPreferences;
```

Comment:
It is the defaults that are passed in, so we need to clone. But it should be done in a structured fashion upstream, so that we know we are safe. Not to rely on the individual APIs doing the correct thing

Reply:
I've changed the code that both RoutingRequest and RoutingPreferences are cloned